examples/gemma/transformer.py

```diff
@@ -128,30 +128,31 @@ def gemma_7b(cls):
     )
 
   @classmethod
-  def gemma_27b(cls):
-    num_layers = 46
+  def gemma2_2b(cls):
```

Comment:
Why don't we add a new method for `gemma_27b` configuration instead of removing the `gemma2_2b` one?

Reply:
They are both in the file - it's just the git that didn't pick up the diff properly.

But also, it makes sense to rename the different 'generations' of gemma, gemma2, gemma3, etc  as separate classes, rather than relying on implicit knowledge about which size came from where.

Moreover, the gemma2_27 didn't have the right normalisation in the attention - will need a separate fix.

I also see that the Google-generated PR adopted the same fix as mine to the Block module.  Good for you!